src/rqt_plot/plot_widget.py

```diff
@@ -114,6 +115,14 @@ def get_plot_fields(node, topic_name):
         slot_type, slot_is_array, array_size = _parse_type(slot_type)
         is_array = slot_is_array and field_index is None
 
+        # If the field is an array, we need to update slot_type to extract
+        # the inner type, e.g. sequence<foo_msgs/Foo> -> foo_msgs/Foo
+        # See https://design.ros2.org/articles/mapping_dds_types.html
+        if is_array:
+            sequence_inner_type_regexp = re.match('sequence<(.*)>', slot_type)
+            if sequence_inner_type_regexp:
+                slot_type = sequence_innter_type_regexp.groupdict()['slot_type']
```

Comment:
This looks like a spelling mistake. Please update and afterward test your pull request locally.